packages/core/src/generators/vue.ts

```diff
@@ -362,6 +362,9 @@ export const blockToVue: BlockRenderer = (node, options, scope) => {
   }
 
   if (node.bindings._text?.code) {
+    const slotPrefixPhrase = 'props.slot';
+    if (node.bindings._text?.code?.substring(0, slotPrefixPhrase.length) === slotPrefixPhrase)
+      return `<slot name="${node.bindings._text?.code?.substring(slotPrefixPhrase.length).toLowerCase()}"/>`;
     return `{{${stripStateAndPropsRefs(node.bindings._text.code as string)}}}`;
```

Comment:
One small thing: we have `isSlotProperty` that you can reuse, and `stripStateAndPropsRefs` will remove `props.`. So I recommend an approach like this one:

```suggestion
  const textCode = node.bindings._text?.code;
  if (textCode) {
    const strippedTextCode = stripStateAndPropsRefs(textCode);
    if (isSlotProperty(strippedTextCode)) {
      return `<slot name="${textCode.substring('slot'.length).toLowerCase()}"/>`;
    }
    return `{{${strippedTextCode}}}`;
  }
```

We can actually also export SLOT_PREFIX from https://github.com/BuilderIO/mitosis/blob/78a6d882cd6a4a022d5910831d71b194a5b30c1d/packages/core/src/helpers/slots.ts#L1 to reuse it here, or add a `stripSlotPrefix` helper to the `slots.ts` helper and import it here :)

Reply:
@samijaber thank you for your suggestions, code looks much better, I just simply wasn't aware of those utils. Nevertheless I've added `stripSlotPrefix` utils inside `slots.ts` helper file